client/lib/index.js

```diff
@@ -287,14 +287,18 @@ module.exports = class Client extends PassThrough {
 	run() {
 		const main = async (deviceType, suite, conf, image) => {
 			process.on('SIGINT', async () => {
-				await rp.post(`${this.uri.href}stop`).catch(this.log.bind(this));
+				await rp.post(`${this.coreUrl}stop`).catch(this.log.bind(this));
 				process.exit(128 + constants.signals.SIGINT);
 			});
 			process.on('SIGTERM', async () => {
-				await rp.post(`${this.uri.href}stop`).catch(this.log.bind(this));
+				await rp.post(`${this.coreUrl}stop`).catch(this.log.bind(this));
 				process.exit(128 + constants.signals.SIGTERM);
 			});
 
+			const heartbeat = setInterval(async() => {
+				await rp.get(`${this.uri}/heartbeat`);
+			}, 1000*20);
```

Comment:
nitpick, but in a future PR we should make this configurable

Reply:
agreed